src/reducers/reducer.js

```diff
@@ -1073,12 +1072,11 @@ export const onSetWCConnectionFailed = (state, { payload }) => ({
   },
 });
 
-const saga = createSagaMiddleware();
-const middlewares = [
-  saga,
-  thunk,
-];
-
-export const store = createStore(reducer, applyMiddleware(...middlewares));
-
-saga.run(rootSagas);
+/**
+ * @param {Object} action.payload The network settings emitted in saga
+ * @see updateNetworkSettings
+ */
+export const onNetworkSettingsUpdateSucess = (state, { payload }) => ({
```

Comment:
```suggestion
export const onNetworkSettingsUpdateSuccess = (state, { payload }) => ({
```

Reply:
I have fixed it in the PR(15), see: https://github.com/HathorNetwork/hathor-wallet-mobile/pull/360/commits/c7fed45c31a60a05af49d2001845ce5ff8726802#diff-44fa50eab045a7af7a0046e496ee4142690d9f81a36393055f2e8d606b22df40